package.json

```diff
@@ -755,6 +757,7 @@
         "powershell.developer.powerShellExePath": {
           "type": "string",
           "default": "",
+          "scope": "machine",
           "isExecutable": true,
           "description": "Deprecated. Please use the 'powershell.powerShellExePath' setting instead"
```

Comment:
Maybe we should get rid of this setting?

Reply:
Agreed

Reply:
done